src/NativeScript/Metadata/Metadata.mm

```diff
@@ -207,8 +207,10 @@ int compareIdentifiers(const char* nullTerminated, const char* notNullTerminated
     } while (this->_topLevelIndex < this->_globalTable->buckets.count);
 }
 
+extern char startOfMetadataSection __asm("section$start$__DATA$__TNSMetadata");
```

Comment:
Does this work when the runtime is dynamically linked into the application? Also, we should try to stay away from ASM what with bitcode and everything. The mach-o APIs to resolve sections in the main binary didn't work out?